devel/hwloc/Portfile

```diff
@@ -3,8 +3,8 @@
 PortSystem          1.0
 
 name                hwloc
-set branch          1.11
-version             ${branch}.8
+set branch          2.1
+version             ${branch}.0
```

Comment:
Since this is a major version number increase, have you verified that the ports that depend on hwloc still work with this version?

Reply:
How to get a list of ports depend on hwloc? On my system nothing depends on it

Reply:
`port echo depends:hwloc`

Reply:
Thanks, and that's a lot of ports... I might just submit this as a new port later, since 1.x is still offered as 'ultrastable' on the official site.